indexing-service/src/main/java/org/apache/druid/indexing/overlord/TaskMaster.java

```diff
@@ -136,6 +136,9 @@ public void becomeLeader()
           }
 
           leaderLifecycle.addManagedInstance(taskRunner);
+          for (TaskRunner subTaskRunner : taskRunner.getSubTaskRunners()) {
+            leaderLifecycle.addManagedInstance(subTaskRunner);
+          }
```

Comment:
Is this change needed if `KubernetesAndWorkerTaskRunner#start` and `KubernetesAndWorkerTaskRunner#stop` call start and stop on the task runners they are delegating to?